features/admin/manage_users.feature

```diff
@@ -0,0 +1,27 @@
+@manage-users
+Feature: Admin manager can manage users
+
+Scenario: Admin Manager user can log in and suspend admin users
+  Given I am logged in as the production admin-manager user
+  And I am on the /admin page
+  And I click the 'Manage users' link
+  #TODO change this email address
```

Comment:
Do we need to do this TODO before anything here will work?

Reply:
Removed